interface-definitions/system-option.xml.in

```diff
@@ -117,6 +117,26 @@
                <valueless/>
              </properties>
            </leafNode>
+           <leafNode name="root-partition-auto-resize">
+             <properties>
+               <help>Enable/disable root partition autoextension while system boot</help>
+                 <completionHelp>
+                   <list>enable disable</list>
```

Comment:
We have had a policy against this kind of syntax for a long while. If you see a node whose only values are "enable/disable", "true/false" or similar, it's a legacy that should eventually be converted.

The reason is that the value is just a noise. `set system options root-partition-auto-resize` says everything, and "enable" after it adds nothing new. Moreover, a node with "enable/disable" values has two states with the same effect: "exists with default value" and "does not exist", which complicates the script logic.

I can see where you likely got the idea though: `reboot-on-panic` still uses a "true/false" syntax. It's a very old option though and we just never got to updating its syntax (though we should).

This option should be a valueless node like `beep-when-fully-booted`.

Reply:
Got it. Fully agree with you. I'll change it.